problems.rb

```diff
@@ -0,0 +1,9 @@
+#TODO: fix this whole fuction
+def hithisfuncissocomp!!licated(n)
+    for i in 0..n
```

Comment:
This loop is from 0 to n, which means we are looping for n+1 times, which is 1 more than we need. Better still, we could do a for loop from 0 to 4n-1, which solves the whole issue.